mysql-test/suite/tokudb/r/cluster_key_part.result

```diff
@@ -2,60 +2,44 @@ set default_storage_engine='tokudb';
 DROP TABLE IF EXISTS t;
 CREATE TABLE t (a INT NOT NULL AUTO_INCREMENT, b INT, PRIMARY KEY(a), CLUSTERING KEY b(b)) ENGINE=TokuDB
 PARTITION BY RANGE(a) (PARTITION p0 VALUES LESS THAN (100) ENGINE = TokuDB, PARTITION p2 VALUES LESS THAN MAXVALUE ENGINE = TokuDB);
-Warnings:
-Warning	1287	The partition engine, used by table 'test.t', is deprecated and will be removed in a future release. Please use native partitioning instead.
 SHOW CREATE TABLE t;
 Table	Create Table
 t	CREATE TABLE `t` (
   `a` int(11) NOT NULL AUTO_INCREMENT,
   `b` int(11) DEFAULT NULL,
   PRIMARY KEY (`a`),
-  CLUSTERING KEY `b` (`b`)
-) ENGINE=TokuDB DEFAULT CHARSET=latin1
-/*!50100 PARTITION BY RANGE (a)
+  KEY `b` (`b`)
+) ENGINE=TokuDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci
+/*!50100 PARTITION BY RANGE (`a`)
```

Comment:
This needs rebased to PS 8.0 HEAD and re-recording as clustering keys are now properly supported.

Reply:
Rebased and re-recorded.